modules/claims_api/spec/requests/v1/forms/526_spec.rb

```diff
@@ -872,6 +872,44 @@
               end
             end
           end
+
+          context "when 'unitName' is empty" do
+            let(:unit_name) { '' }
+
+            it 'returns a successful response' do
```

Comment:
Should we also assert that the unit name transformed into a single space (`' '`)?

Reply:
@tycol7 How would you do that? The unitName is not returned in the response.

Reply:
We could lookup the claim by the id in the response, then check the form data:

```rb
context "when 'unitName' is empty" do
  let(:unit_name) { '' }

  it 'returns a successful response and transforms empty string to single space' do
    mock_acg(scopes) do |auth_header|
      VCR.use_cassette('claims_api/bgs/claims/claims') do
        VCR.use_cassette('claims_api/brd/countries') do
          json_data = JSON.parse data
          params = json_data
          params['data']['attributes']['serviceInformation']['reservesNationalGuardService']['unitName'] = unit_name

          post path, params: params.to_json, headers: headers.merge(auth_header)
          
          expect(response).to have_http_status(:ok)
          
          claim_id = JSON.parse(response.body)['data']['id']
          claim = ClaimsApi::AutoEstablishedClaim.find(claim_id)
          expect(claim.form_data['serviceInformation']['reservesNationalGuardService']['unitName']).to eq(' ')
        end
      end
    end
  end
end
```